### PR TITLE
Archive post / Add tags to Post 

### PR DESCRIPTION
**Pull request by AnuragSahu11**
- Added Archive post functionality
- Users can add/remove tags from there post.
- When these Tags are clicked all post containing that tag will be shown.
- Fixed minor bugs

### CODE REVIEW

File: src/utils/filter-sort-post.jsx [hunk 1]
```diff
@@ -2,33 +2,61 @@ import { filterConstants } from "./constants";
...
+  //   return archiveFilterArr.filter((post) => post.postByID === userID);
+  // }
+  // if (bookmark)
+  //   return archiveFilterArr.filter((post) => bookmark.includes(post.postID));
+
+  // if (feed) {
+  //   return archiveFilterArr.filter(
+  //     (post) => feed.includes(post.postByID) || post.postByID === userID
+  //   );
+  // }
```
Comment (abhij1607) [inline src/utils/filter-sort-post.jsx:58]: remove this if not required

**Review by abhij1607 — APPROVED** [review]
Neat & clean code
GW implementing tags

Can be merged

File: src/utils/filter-sort-post.jsx (hunk 1 above)
Comment (AnuragSahu11) [reply src/utils/filter-sort-post.jsx:58]: Thanks for mentioning completely forgot about it.

**Review by rishichourasia — APPROVED** [review]
functionality working as intended, remove comments as said so . can be merged ,

**Pull request merged**